cs.html

```diff
@@ -752,29 +752,43 @@ <h2>Constructors</h2>
             </li>
             <li>Run these steps in parallel:
               <ol>
-                
-                <!-- TODO: Figure out a way to convey the origin in an encrypted manner
-                           and then use a non-empty value like so:
-                Let <var>serializedOrigin<var> be the empty string ,<code>""</code>.
-                Let <var>serializedOrigin<var> be <var>quictransport<var>'s 
-                <a href="https://html.spec.whatwg.org/multipage/webappapis.html#relevant-settings-object">relevant settings object</a>'s
-                <a href="https://html.spec.whatwg.org/multipage/webappapis.html#concept-settings-object-origin">origin</a>,
-                <a href="https://html.spec.whatwg.org/multipage/origin.html#ascii-serialisation-of-an-origin">serialized</a>.
-                The user agent MUST include a QUIC transport parameter with ID of web_origin(0x3330) 
-                and value of <var>serializedOrigin</var>.
-                -->
                 <li>Establish a QUIC connection to the address identified by the
-                <var>parsedUrl</var>'s host and port.  
-                During connection establishment, use of this API must be indicated 
-                by selecting the ALPN [[!ALPN]] token "wq" in the crypto handshake.
-                <!-- TODO: register "wq" with IANA. -->  
+                  <var>parsedUrl</var>'s host and port.  
+                  During connection establishment, use of this API must be indicated 
+                  by selecting the ALPN [[!ALPN]] token "wq" in the crypto handshake 
+                  and including a QUIC transport parameter with ID web_client(0x333C)
+                  and empty value.
+                  <!-- TODO: register "wq" with IANA. --> 
+                </li>
+                <li>If the connection fails, set <var>quictransport</var>'s <a>[[\QuicTransportState]]</a>
+                    internal slot to <code>"failed"</code> and abort these steps.
+                </li>
+                <li>Let <var>joinedAcceptedOrigins</var> be the QUIC transport parameter provided
+                    by the server with ID web_accepted_origins(0x333A).  If the transport parameter is
+                    absent, set <var>quictransport</var>'s <a>[[\QuicTransportState]]</a>
+                    internal slot to <code>"failed"</code> and abort these steps.
+                    <!-- TODO: register 0x333A with IANA -->
+                </li>
+                <li>
+                    Let <var>serialiedAcceptedOrigins</var> be
```

Comment:
serialiedAcceptedOrigins --> "serializedAcceptedOrigins"

Reply:
Fixed.  Thanks for catching